builds/e2e/e2e-release.yaml

```diff
@@ -30,8 +30,8 @@ stages:
         variables:
           os: linux
           arch: arm32v7
-          artifactName: iotedged-debian9-arm32v7
-          identityServiceArtifactName: packages_debian-9-slim_arm32v7
+          artifactName: iotedged-debian10-arm32v7
+          identityServiceArtifactName: packages_debian-10-slim_arm32v7
```

Comment:
Why did we pick debian 10 vs 11?

Reply:
Because I just didn't know. I thought of it as a sliding window (in terms of checking for backward compatibility).

Damon pointed out the same thing. I'm changing to 11.